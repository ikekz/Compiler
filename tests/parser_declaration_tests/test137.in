function f(a: integer) : integer;
var
	v: integer;
begin
end;
var
	a, v: integer;
begin
end.