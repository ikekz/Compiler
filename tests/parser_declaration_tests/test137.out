integer        type           integer
double         type           double
char           type           char
f              function       return type: 
                              integer
                              args:
                              a              valueparam     integer
                              locals:
                              v              var            integer        0
a              var            integer        0
v              var            integer        0
